src/components/search-input/search-input.js

```diff
@@ -7,30 +7,48 @@ export const SearchInput = ({
   hasIcon = true,
   onValueChange,
   placeholder = "cauta",
-  value
+  value,
+  onClick,
```

Comment:
I'm not sure we need `onClick`. I wouldn't see a case where we would click on the input field. It's common to have a button next to it if needed, in which case the `onClick` will be on the button and not the input field.

Reply:
I mean, it doesn't hurt, but it looks like an anti-pattern to me.

Reply:
we do have an icon that can be clicked

Reply:
the onClick is't not actually for the input, it's for the SearchIcon
would it be a good ideea to merge both onClick and onEnter intro one prop "onSearch" ?

Reply:
You mean the search icon? It's still an anti-pattern. I haven't seen a case where you would click an icon in an input in a long time.

Reply:
Will get this in the `v1.0.4` release. We can update it later if needed. 🙌